Improve release docs (#677)
tektoncd/cli: RELEASE_PROCESS.md

dnf upgrade tektoncd-cli
```

> - Make an update for [homebrew-core](https://github.com/Homebrew/homebrew-core/blob/master/Formula/tektoncd-cli.rb) for tektoncd-cli formula.

Reviewer: s/for/on/

Author: done

>   * Make sure you get the proper `sha256sum` You need to download the `Source Code (tar.gz)` from the release and hit below command for the respective release

Reviewer 2: ```suggestion
  * Make sure you get the proper `sha256sum`. You need to download the `Darwin_x86_64.tar.gz` from the [release section of the cli GitHub](https://github.com/tektoncd/cli/releases) and run the command below for the release version you are working on:
```

Author: But I have addressed the other changes, but it is Source Code tarball not Darwin

Reviewer 2: I guess it can be either, so works for me.

Author: No, Darwin has different `sha` and CI failed for that

  
    ```shell script

>     openssl dgst --sha256 cli-0.7.1.tar.gz 

Reviewer: I guess openssl works but so should be the `sha256sum`  tool

Author: yaa both of them works

Reviewer 2: Maybe more generally to get sha256 and say the example below uses `openssl`? I'm not sure which too is more commonly used.

Author: i have changed the command to `sha256sum`

    ```
    

>     You will get a output like

Reviewer 2: ```suggestion
    You will get similar output to what is shown below:
```

    
    `SHA256(cli-0.7.1.tar.gz)= 72df3075303d2b0393bae9a0f9e9b8441060b8a4db57e613ba8f1bfda03809b5`

>   * Raise a PR to update like [this](https://github.com/Homebrew/homebrew-core/pull/46492)

Reviewer 2: ```suggestion
  * Raise a PR to update like [this](https://github.com/Homebrew/homebrew-core/pull/46492) to Homebrew Core
```

Author: done
